control-plane-operator/controllers/hostedcontrolplane/olm/assets/packageserver-deployment.yaml

```diff
@@ -16,11 +16,6 @@ spec:
     type: RollingUpdate
   template:
     metadata:
-      annotations:
-        include.release.openshift.io/self-managed-high-availability: "true"
-{{ if .RestartDate }}
-        openshift.io/restartedAt: "{{ .RestartDate }}"
-{{ end }}
       labels:
         app: packageserver
```

Comment:
Interesting, was there a reason we decided to remove the labels from the catalog and olm deployments but not the packageserver deployment?

Reply:
No reason, will remove. The reason that I removed it from the other ones is that we're now using the manifests for the spec of the resource. Nothing in the metadata is read from here.

Reply:
Actually nevermind, this is part of the deployment template. The labels are needed here for the match the deployment selector.

Reply:
Ah my mistake, I thought this was the `metadata.labels` field.